reference/locations.v3.yml

```diff
@@ -1731,6 +1735,38 @@ components:
         enum:
           - asc
           - desc
+    date_created_min:
+      name: 'date_created:min'
+      description: |
+        'Query parameter that lets you filter by the minimum date created, for example, `2024-05-14T09:34:00Z`. Returns metafields created after this date.'
```

Comment:
Do we want to mention that we can provide just a date as well ?

Reply:
We can, are you suggesting that we add a `date_created` parameter to the list? Would I need to do this just for the locations file or for the other APIs in [PR 322](https://github.com/bigcommerce/docs/pull/322/files).